js/binancetr.js

```diff
@@ -0,0 +1,308 @@
+            'has': {
+                'fetchMarkets': true,
+                'fetchOrderBook': true,
+                'fetchTime': true,
+                'fetchTrades': true,
+            },
+            'timeframes': {
+                '1m': '1m',
+                '3m': '3m',
+                '5m': '5m',
+                '15m': '15m',
+                '30m': '30m',
+                '1h': '1h',
+                '2h': '2h',
+                '4h': '4h',
+                '6h': '6h',
+                '8h': '8h',
+                '1d': '1d',
+                '3d': '3d',
+                '1w': '1w',
+                '1M': '1M',
+            },
+            'urls': {
+                'logo': '',
+                'api': {
+                    'public': 'https://www.trbinance.com/open/',
+                    'public3': 'https://api.binance.me/api/v3/',
+                    'private': 'https://www.trbinance.com/open/',
+                },
+                'www': 'https://www.trbinance.com',
+                'doc': [
+                    'https://www.trbinance.com/apidocs/',
+                ],
+                'fees': [
+                    'https://www.trbinance.com/en/fees/schedule',
+                ],
+            },
+            'api': {
+                'public': {
+                    'get': [
+                        'common/time',
+                        'common/symbols',
+                        'market/depth',
+                        'trades',
+                        'aggTrades',
+                        'klines',
+                    ],
+                },
+                'private': {
+                    'get': [
+                        'orders/detail',
+                        'orders',
+                        'account/spot',
+                        'account/spot/asset',
+                        'orders/trades',
+                        'withdraws',
+                        'deposits',
+                        'deposits/address',
+                    ],
+                    'post': [
+                        'orders',
+                        'orders/cancel',
+                        'orders/oco',
+                        'withdraws',
+                    ],
+                },
+            },
+            'fees': {
+                'trading': {
+                    'tierBased': true,
+                    'percentage': true,
+                    'maker': 0.1 / 100,
+                    'taker': 0.1 / 100,
+                },
+            },
+            'options': {
+                'networks': {
+                },
+            },
+            'commonCurrencies': {
+            },
+            'exceptions': {
+            },
+        });
+    }
+
+    async fetchTime (params = {}) {
+        const response = await this.publicGetCommonTime (params);
+        //
+        //      {
+        //          "code":0,
+        //          "msg":"Success",
+        //         "data":null,
+        //          "timestamp":1641318638443
```

Comment:
`"data"` seems to be shifted to the left here, missing one space.

Reply:
fixed